project/Helpers.scala

```diff
@@ -15,7 +15,6 @@ object V {
   lazy val scala = "2.13.10"
   lazy val akka = "2.7.0"
   lazy val akkaHttp = "10.5.0"
-  lazy val akkaGrpc = "2.2.1"
```

Comment:
Removed this because it was not being used as a variable

Reply:
OK